docs/metodologia.md

```diff
@@ -0,0 +1,192 @@
+# Metodologia
+
+### Metodologia adotada no projeto
+<p align="justify">&emsp;&emsp;Foi adotado como base o <i>framework scrum</i> onde empregamos diversos processos e técnicas adaptando às necessidades da equipe. </p>
+<p align="justify">&emsp;&emsp;O objetivo do Scrum no projeto é permitir o controle do trabalho a ser realizado por meio de uma gestão dinâmica, assim identificar obstáculos durante o processo de desenvolvimento e reagir a eles. A abordagem iterativa e incremental empregada otimiza a previsão e monitoramento de riscos </p>
+
+### Papeis
+|Papel|Membro|
+|-|-|
+| Scrum Master | Isaque Alves |
+| Product Owner | João Egewarth |
+| Software Architect | Gabriela Alves |
+| DevOps | Eliseu Egewarth |
+| Developers | Beatriz Hanae, Gabriela Guedes, Vitor Leal, Felipe Campos, Ezequiel Oliveira, Guilherme Deusdara |
+
+#### Scrum Master
+
+- Responsável por garantir que o Time Scrum esteja aderindo aos valores do Scrum, às práticas e às regras;
+- Ajuda o Time Scrum e a organização a adotarem o Scrum;
+- Educa o Time Scrum treinando-o e levando-o a ser mais produtivo e a desenvolver produtos de maior qualidade;
+- Ajuda o Time Scrum a entender e usar autogerenciamento e interdisciplinaridade;
+- Resolver impedimentos;
+- Fazer a gestão de riscos e custos;
+- Documentar as sprints;
+- Preparar as reuniões e conduzir-las para que não passe do tempo determinado.
+- Não gerencia o Time Scrum; o Time Scrum é auto-organizável.
+
+#### Product Owner
+
+- Responsável pelo gerenciamento do Backlog do Produto e por garantir o valor do trabalho realizado pelo Time.
+- Manter o Backlog do Produto e garante que ele está visível para todos.
+- Informar a todos quais itens têm a maior prioridade, de forma que todos sabem em que se irá trabalhar.
+- Definir e priorizar os itens do Backlog do Produto.
+- Somente o PO tem a permissão de dizer ao Time para
+trabalhar em um outro conjunto de prioridades.
+- As decisões do Product Owner devem ser visíveis no conteúdo e na priorização do Backlog do Produto.
+
+#### DevOps
+- Garantir que todas as alterações de código e configurações sejam feitas usando mecanismos automatizados, rastreáveis ​​e repetíveis.
+- Automatizar a Infraestrutura.
+- Automatizar e garantir a integração continua
+- Autmatizar e garantir o deploy continuo
+- Facilitar o processo de desenvolvimento
+- Organizar os pipeline do produto.
+
+#### Software Architect
+- Elaborar e projetar a melhor solução técnica a ser adotada no projeto
+- Garantir que a arquitetura projetada seja implementada
+- Cuidar para que o produto atenda ás expectivas do cliente dentro do prazo e custos definidos
```

Comment:
acho que poderia mostrar/dar meios de como implementar a arquitetura tbm, algo assim.

Reply:
não to conseguindo achar palavras...